tests/test_permissions.py

```diff
@@ -1,3 +1,5 @@
+from contextlib import contextmanager
```

Comment:
We cvan't use `from contextlib import nullcontext as does_not_raise` because you are using it in a context manager, right?

Reply:
You're right, I saw this on a forum and was not sure about the `import  as does_not_raise` but didn't see it on the official doc on Python, let's use that instead !